app/views/accounts/_emails.html.erb

```diff
@@ -15,7 +15,16 @@
             <%= email.email %>
           </span>
           <% unless email.confirmed_at %>
-            &nbsp;<%= t('email_addresses.unconfirmed') %>
+            <% if IdentityConfig.store.dexcom_skip_email_confirmation %>
+              <%
+                # Dexcom Change
+                # This is a shortcut to quickly confirm the email. This is being used for testing
+                # so this is a workaround to avoid sending an email.
+              %>
+              &nbsp;(<%= link_to('Confirm Email',add_email_confirmation_url(confirmation_token: email.confirmation_token, locale: locale_url_param)) %>)
+            <% else %>
+              &nbsp;<%= t('email_addresses.unconfirmed') %>
+            <% end %>
```

Comment:
This was an affordance in case you screwed up the previous step. It gives you a little link to confirm it:

![image](https://github.com/user-attachments/assets/8a917fb2-3961-4d2d-8bc7-62908ebd4358)